zenodo_accessrequests/models.py

```diff
@@ -134,7 +136,7 @@ def validate_token(cls, token, expected_data):
     def query_by_owner(cls, user):
         """Get secret links by user."""
         return cls.query.filter_by(
-            owner_user_id=user.get_id()
```

Comment:
I noted a "strange" behavior, because it returns `unicode` instead of `integer`.
So, when you try to read the value of `owner_user_id`, it'll be e.g. `u"1"` instead of `1`.